eva/udfs/ndarray/timestamp.py

```diff
@@ -0,0 +1,49 @@
+# coding=utf-8
+# Copyright 2018-2022 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import time
+
+import pandas as pd
+
+from eva.udfs.abstract.abstract_udf import AbstractUDF
+
+
+class Timestamp(AbstractUDF):
+    @property
+    def name(self) -> str:
+        return "Timestamp"
+
+    def setup(self):
+        pass
+
+    def forward(self, inp: pd.DataFrame) -> pd.DataFrame:
+        """
+        inp: DataFrame -> out: DataFrame
+            second           timestamp
+        0   int           0   string
+        1   int           1   string
+        """
+
+        # Sanity check
+        if len(inp.columns) != 1:
+            raise ValueError("input must only contain one column (seconds)")
+
+        seconds = pd.DataFrame(inp[inp.columns[0]])
```

Comment:
Isn't it `no-op`?

Reply:
This is a timestamp UDF.

Reply:
`SELECT id, seconds, Timestamp(seconds)
                          FROM MyVideo
                          WHERE Timestamp(seconds) <= "00:00:01";`